debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java

```diff
@@ -87,6 +88,7 @@ public class MySqlStreamingChangeEventSource implements StreamingChangeEventSour
     private long initialEventsToSkip = 0L;
     private boolean skipEvent = false;
     private boolean ignoreDmlEventByGtidSource = false;
+    private boolean isGtidModeEnabled = false;
```

Comment:
If you move this to an instance variable,  it will make more sense to initialize this value in the constructor and mark it as `final` as this shouldn't change over the runtime of the connector.